acceptance/tests/modules/install/long_path_name.rb

```diff
@@ -0,0 +1,34 @@
+test_name "puppet module install (long file path)"
+require 'puppet/acceptance/module_utils'
+extend Puppet::Acceptance::ModuleUtils
+
+confine :except, :platform => /centos-4|el-4/ # PUP-5226
+confine :except, :platform => /aix/ # PUP-5501
+
```

Comment:
oh right.  i'm pretty sure the aix confine in the other tests is mine. 
if they all do this, because... pmt, could probably use an abstraction across tests.  oh well.  another ticket